packages/kaizen-tg/_templates/drupal-9-theme/new/packages/components/_variables.css

```diff
@@ -2,10 +2,56 @@
 to: <%= h.src() %>/<%= h.changeCase.lower(name) %>/packages/components/_variables.css
 ---
 :root {
-  --root-font-size: 2vw;
-  --base-font-size: 16px;
+  /* font-sizes */
+  --font-size--xxxxxl: 48px;
+  --font-size--xxxxl: 40px;
+  --font-size--xxxl: 36px;
+  --font-size--xxl: 30px;
+  --font-size--xl: 24px;
+  --font-size--m: 20px;
+  --font-size--l: 18px;
+  --font-size--base: 16px;
+  --font-size--s: 14px;
+  --font-size--xs: 12px;
+  --font-size--xxs: 10px;
 
+  /* outline settings */
+  --outline-color: blue;
+  --outline-size: 2px;
+  --outline-offset: var(--outline-size);
+
+  /* site sizes */
+  --site-max-width: 1920px;
+
+  /*
+   * Root font size changes according to the breakpoints
+   * Mobile design width is 375px, so everything below should be scaled proportionally
+   * font would be decreased
+  */
+  --root-font-size: 4.278vw;
+
+  /* stylelint-disable-next-line order/order */
+  @media all and (min-width: 376px) {
+    --root-font-size: 100%;
```

Comment:
it's a project specific settings, shouldn't be here

Reply:
^ prev comment